### ISSUE TITLE
Issue withe mapbox geocoder documentation for web development 

### ISSUE THREAD

**Issue by Chris8800** [mapbox/mapbox-unity-sdk#1371]
Hello mapbox team,

I have been working with the search products of mapbox. The documentation and examples of the mapbox geocoding is very cut off and hard to follow! Documentations on web development is absolutely not easy to find on your site and has no understandable starting point for newcomers. I am trying to implement your geocoder with java into mvn/springframe. It is close to impossible to find an understandable solution for this issue. I am trying to prevent using the snippets that are all in HTML or defined in JS scripts. I am trying to create an object with a string + access token and pass the values onto the geocoding response. I am afraid that the tutorial expressed on https://docs.mapbox.com/android/java/overview/geocoder/ is not suitable for web developers. There is no other alternative that i see fit for the problem I am dealing with. I need help or an understandable documentation for backend development with java and spring. 

All the best, 
Chris

**Comment by brnkhy** [comment]
Hey @Chris8800, this is the unity sdk repo so admittedly I don't know much about other frameworks but I'll ping @jordy-isaac and he'll probably know who to relay your feedback.

**Comment by Chris8800** [comment]
Hey @brnkhy , thanks for the reply. I have found a way to implement it, but it is not a very stable and secure way for a longterm purpose. I am working with springboot/mvn on the backend and angular on the frontend. I am trying to keep angular as clean as possible and have all the working stuff in the backend. I have the project still web based. The android pack is not going to work with my mvn pom and I have a hard time finding the documentation for web devs with java and frameworks like spring. Best wishes, Chris

**Comment by brnkhy** [comment]
@Chris8800 admittedly I'm not one bit familiar with those frameworks and libraries so I don't get it but happy to hear at least you have a temporary solution. :D 
I'm closing this for now but I would suggest mailing support as they would relay it to android (or web?) team right away.
